charon.py

```diff
@@ -1,224 +1,39 @@
 # charon.py
+import logging
 import os
+import random
+
 import discord
-from discord.ext import tasks, commands
+from discord.ext import commands
 from dotenv import load_dotenv
-import roles
-import party
-import random
-import logging
 
-load_dotenv()
-TOKEN = os.getenv('DISCORD_TOKEN')
+from cogs.roles.roles_cog import RolesCog
+from cogs.party.party_cog import PartyCog
 
-COMMAND_PREFIX = '.'
-bot = commands.Bot(command_prefix=COMMAND_PREFIX)
 
-LFG_CHANNEL = 'lookingforgroup'
+load_dotenv()
+TOKEN = os.getenv('DISCORD_TOKEN')
+COMMAND_PREFIX = os.getenv('COMMAND_PREFIX')
+BACKGROUND_LOOP_TIME = os.getenv('BACKGROUND_LOOP_TIME')
```

Comment:
BACKGROUND_LOOP_TIME does not get used in charon.py; we should probably remove it